flow/scheduling/torque.py

```diff
@@ -146,9 +146,12 @@ def submit(self, script, after=None, pretend=False, hold=False, flags=None, *arg
             with tempfile.NamedTemporaryFile() as tmp_submit_script:
                 tmp_submit_script.write(str(script).encode('utf-8'))
                 tmp_submit_script.flush()
-                output = subprocess.check_output(
-                    submit_cmd + [tmp_submit_script.name])
-            jobsid = output.decode('utf-8').strip()
+                try:
+                    output = subprocess.check_output(
+                        submit_cmd + [tmp_submit_script.name])
+                    jobsid = output.decode('utf-8').strip()
+                except subprocess.CalledProcessError as e:
+                    print(e.output())
```

Comment:
Reviewing this, I realized that we already implement the desired behavior for the SLURM scheduler, just not for TORQUE. Thank you very much for fixing this!

Can you update this line to raise a `SubmitError` similar to how we do this for the SLURM scheduler?
https://github.com/glotzerlab/signac-flow/blob/0685611bfa370ab47166e80449e6de6aaece443f/flow/scheduling/slurm.py#L138

In this case it should say "qsub" instead of "sbatch".